src/buffer_par.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(exported_private_dependencies)]
```

Comment:
Please avoid changes unrelated to the blur algorithm. The issue with a false positive lint is known and also happens on the main branch. We will take care of it separately, and CI failing on this issue is not a blocker for merging this PR.

Reply:
I was not sure about the policy but I simply dropped the commit and the other unrelated clippy failure.